src/Uno.UI/Microsoft/UI/Xaml/Controls/NavigationView/NavigationViewItem.cs

```diff
@@ -1106,14 +1106,7 @@ private void ProcessPointerCanceled(PointerRoutedEventArgs args)
 			_uno_pointerDeferring?.Stop();
 
 			m_isPressed = false;
-			// m_isPointerOver should be true before this event so this doesn't need to be set to true in the else block...
-			// What this flag tracks is complicated because of the NavigationView sub items and the m_capturedPointers that are being tracked..
-			// We do this check because PointerCaptureLost can sometimes take the place of PointerReleased events.
-			// In these cases we need to test if the pointer is over the item to maintain the proper state.
-			if (IsOutOfControlBounds(args.GetCurrentPoint(this).Position))
-			{
-				m_isPointerOver = false;
-			}
+			m_isPointerOver = false;
```

Comment:
@MartinZikmund thoughts?

Reply:
Was the comment ours?

Reply:
It was coming from MUX, but maybe it was just to handle some UWP oddity

Reply:
The code is originally from WinUI sources.

https://github.com/microsoft/microsoft-ui-xaml/blob/210349d9d1e0aeb8ee144ec027071a68ac578bdb/dev/NavigationView/NavigationViewItem.cpp#L1077-L1085

I think we should comment the condition with "Uno docs" comment.

Reply:
It has been added 2 years ago https://github.com/microsoft/microsoft-ui-xaml/commit/c27a05caa0eeebaacdbd2106aebd12a6fc3dd912, most probably for the "Fix Animated chevron not working for heirarchical nav"

Reply:
As it's from the WinUI repro, I'll restore the code and add a `#if false` to ignore it, and we will check in canaries.